### PR TITLE
Update DmaDriver.h

### PR DESCRIPTION
**Pull request by ruck314**
### Description
- Applying the linter fix

### CODE REVIEW

**Review by JJL772 — APPROVED** [review]
Linter seems unhappy in a few files due to `#include`'s

**Comment by ruck314** [comment]
@JJL772 Yeah, still working through them

File: common/app_lib/PrbsData.cpp [hunk 1]
```diff
@@ -22,6 +22,8 @@
 #include <stdlib.h>
 #include <stdio.h>
```
Comment (bengineerd) [inline common/app_lib/PrbsData.cpp:23]: I think we need to include either <stdio.h> or <stdio> but not both.

**Comment by JJL772** [comment]
Oops, did not realize this was a draft!

**Pull request merged**